modules/layers/src/point-cloud-layer/point-cloud-layer.wgsl.ts

```diff
@@ -0,0 +1,107 @@
+// deck.gl
+// SPDX-License-Identifier: MIT
+// Copyright (c) vis.gl contributors
+
+export default /* wgsl */ `\
+// TODO(ibgreen): Hack for Layer uniforms (move to new "color" module?)
+
```

Comment:
@ibgreen any reason why WGSL cannot be added to `layer-uniforms.ts`?

Reply:
I suppose they could. I noticed that the only layer uniform we have is opacity, and given that we need to add support for premultiplied colors, so we likely need a `color` module that would be a natural home for an opacity uniforms, so I felt that might replace layer uniforms.